### ISSUE TITLE
[my-product] - 

### ISSUE THREAD

**Issue by mahmoodjq** [inspireui/support#2582]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - Try to explain your all problems in one ticket, instead of creating multi tickets.
> - After creating an issue, please take the screenshot of http://codecanyon.net/download - to let us know you have purchased our product, or quick note as a comment that you have created this ticket, example: https://tppr.me/DJA8L
> - IMPORTANT: We will close the ticket if you could not prove as our customer.

---------

1. My issue: HOW TO RUN THE FLUX BUILDER ON WINDOWS. I WANT TO CHANGE APP ICON, SPLASH SCREEN, API SETTINGS OF MY WOOCOMMERECE WEBSITE! I HAVE DONE ALL THE FLUTTER INSTALLATION PROCESS BUT COUDLN'T YOUR APP.

2. Screenshots issues (required):

3. Proof of my purchased (required): verified 👍

3. Config files (required)

**Comment by minhcasi** [comment]
@mahmoodjq - the current release of fluxbuilder could not change app icon / splash screen - we will investigate to update it soon

In the meantime please check this document guide - https://docs.inspireui.com/fluxstore/customization/#2-logo-splash-screen 

You can download the window version on this link https://github.com/inspireui/fluxbuilder/releases/download/1.0.0/FluxBuilder.1.0.zip
and https://github.com/inspireui/fluxbuilder/releases/download/1.0.0/vc_redist.x64.exe

**Comment by minhcasi** [comment]
pls redownload the window build and let us know your checking - thank you!

**Comment by mahmoodjq** [comment]
The builder software is not like what you guys promised. I am not an app developer, so this is only for those who have experience in apps dev. Also, I didn't let me copy the config code. I clicked on the copy icon but it didn't copy anything. It is too slow when I linked my Wordpress website. Products take a long time to load. Now, I had to buy another product from a different seller that is easier to make an app and way faster than this. I am asking for refund. I already sent an email to you guys, I hope you guys will respond to it soon. If not, I will be submitting a ticket through canyon code. Thank you.

**Comment by apu27888** [comment]
Mohmoodjq brother.... I think you have made a comment on Becrux React Native Template.... Right???

I have bouught that too and have figured out all of problems and I'm running my App and website nicely!!! Pls contact me if you have any issue... I can help you brother....

**Comment by mahmoodjq** [comment]
Can I have a link to your app on the store or an apk file please!


On May 16, 2020 at 11:51 AM, apu27888 <notifications@github.com> wrote:

Mohmoodjq brother.... I think you have made a comment on Becrux React
Native Template.... Right???

I have bouught that too and have figured out all of problems and I'm
running my App and website nicely!!! Pls contact me if you have any
issue... I can help you brother....

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/inspireui/support/issues/2582#issuecomment-629690669>,
or unsubscribe

**Comment by apu27888** [comment]
Though I just uploaded products on my site... My site address: [](https://fatikshop.com)

Android App Link: [](https://play.google.com/store/apps/details?id=com.rehal.fatikshop)

**Comment by mahmoodjq** [comment]
Unfortunately, the app is also slow using your website. It seems this app
has many programs. The other thing is I am using RTL and they have many
problems with that. They stole $200 from me and they did nothing. I am
reporting that to Paypal including all emails to Paypal. I moved to shopify
because the failed to do fix it and will pay monthly for a company with a
stable app. Thanks for trying to help.


On May 19, 2020 at 10:54 AM, apu27888 <notifications@github.com> wrote:

Though I just uploaded products on my site... My site address:
<https://fatikshop.com>

Android App Link:
<https://play.google.com/store/apps/details?id=com.rehal.fatikshop>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/inspireui/support/issues/2582#issuecomment-630982033>,
or unsubscribe
